src/modeliteration.js

```diff
@@ -130,7 +130,8 @@ define([
                     presence: 'optional',
                     widget: idsQueryDataSourceWidget,
                 }
-            }
+            },
+            noLogicReference: true
```

Comment:
I think this should be temporary until model iteration gets a nice user friendly UI. Then maybe we could extend the logic expression to include the cases/fixtures that it will pull from.